cmd/oci-runtime-tool/generate.go

```diff
@@ -79,6 +79,7 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "process-cap-add", Usage: "add Linux capabilities"},
 	cli.StringSliceFlag{Name: "process-cap-drop", Usage: "drop Linux capabilities"},
 	cli.BoolFlag{Name: "process-cap-drop-all", Usage: "drop all Linux capabilities"},
+	cli.StringFlag{Name: "process-consolesize", Usage: "specifies the console size in characters"},
```

Comment:
I think it's better to add an instance description like "e.g. WIDTH:HEIGHT" at the end.